runtime/smithy-client/common/src/aws/smithy/kotlin/runtime/client/SdkClientOption.kt

```diff
@@ -21,7 +21,7 @@ public object SdkClientOption {
     /**
      * The service name the operation is executed against
      */
-    public val ServiceName: ClientOption<String> = ClientOption("ServiceName")
+    public val ClientName: ClientOption<String> = ClientOption("ClientName")
```

Comment:
nit: I've began namespacing all of the attributes we define e.g. `aws.smithy.kotlin#ClientName`